.github/actions/get_target_branch/action.yaml

```diff
@@ -0,0 +1,24 @@
+name: Get_target_branch
+description: "Gets target branch from the PR description"
+inputs:
+  text:
+    description: "Text from which to extract the target branch"
+    required: true
+outputs:
+  target-branch:
+    description: "Target branch"
+    value: ${{ steps.extract.outputs.target-branch }}
+
+runs:
+  using: composite
+  steps:
+    - name: Extract target branch
+      id: extract
+      shell: bash
+      run: |
+        branch=$(echo "${{ inputs.text }}" | grep -ioP '(?<=\[java|dotnet|python|ruby|php|golang|cpp|agent|nodejs)@.+?\]' | tr -d '[:space:]' || true)
```

Comment:
Could we limit this to supported lang, and explicitly fail if it's not in the supported list? 

I'm 100% someone will try to use one of them, and will struggle to understand why it's not working.